src/pages/item/components/modification/ItemEdit/ItemEditDetail.jsx

```diff
@@ -25,7 +27,9 @@ const ItemEditDetail = ({ itemDetail, editCallback, onCancel }) => {
     <S.Form onSubmit={handleSubmit}>
       <S.FormInnerWrapper>
         <S.LabelInputSet>
-          <S.Span>닉네임</S.Span>
+          <S.Span>
+            닉네임<span style={{ color: "red" }}>*</span>
```

Comment:
```suggestion
            닉네임 <span style={{ color: "red" }}>*</span>
```

Add는 띄워져있구 Edit은 안 띄워져 있는게 띄워진게 더 예뻐서!

Reply:
오... 어케 이렇게 꼼꼼하지;;